project/build.sbt

```diff
@@ -11,10 +11,6 @@ Compile / unmanagedSourceDirectories ++= {
   ).map(dir => root / s"$dir/src/main/scala")
 }
 
-libraryDependencies ++= Seq(
-  "org.eclipse.jgit" % "org.eclipse.jgit.pgm" % "5.5.1.201910021850-r"
-)
-
```

Comment:
really nice to see this resolved.
refs: https://github.com/scala-native/scala-native/issues/1793